newui/docs/README.md

```diff
@@ -0,0 +1,46 @@
+## Site Documentation
```

Comment:
please make sure this file reflects the instructions for the newui and not the old ui